test/scenarios-bootc/c2cc/el102-src@c2cc-dual-stack.sh

```
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+# shellcheck source=test/bin/c2cc_common.sh
+source "${SCRIPTDIR}/c2cc_common.sh"
```

Shellcheck (0.11.0)

[info] Not following: test/bin/c2cc_common.sh was not specified as input (see shellcheck -x).

(SC1091)